code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -611,6 +619,20 @@
 		else if(new_grow)
 			selected.shrink_grow_size = (new_grow/100)
 
+	if(href_list["b_burn_dmg"])
+		var/new_damage = input(user, "Choose the amount of burn damage prey will take per tick. Ranges from 3 to 6.", "Set Belly Burn Damage.") as num|null
+		if(new_damage == null)
+			return
+		Clamp(new_damage, 3, 6)
+		selected.digest_burn = new_damage
+
+	if(href_list["b_brute_damage"])
+		var/new_damage = input(user, "Choose the amount of brute damage prey will take per tick. Ranges from 3 to 6 Numbers below 3 may result in digestion being extremely slow.", "Set Belly Brute Damage.") as num|null
```

Comment:
Still says "Numbers below 3 may result in digestion being extremely slow.", despite being unable to set it below 3.

Reply:
This is why I ask people to review my code.